contrib/hadoop-store-builder/src/java/voldemort/store/readonly/mr/azkaban/gobblin/build.gradle

```diff
@@ -0,0 +1,48 @@
+apply plugin: 'com.github.johnrengelman.shadow'
```

Comment:
Typically, we do not put gradle build files under the src directories. Can we move it into a proper module, at the root of the project?